internal/cloud/realm/client.go

```diff
@@ -31,6 +31,8 @@ type Client interface {
 	DeleteApp(groupID, appID string) error
 	FindApps(filter AppFilter) ([]App, error)
 
+	FindSecrets(app App) ([]Secret, error)
```

Comment:
change this signature to accept `groupID, appID string` (similar to the other method signatures here for this interface)